triple-store/triple-store-api/src/main/java/com/powsybl/triplestore/api/TripleStore.java

```diff
@@ -33,7 +34,14 @@
 
     PropertyBags query(String query);
 
-    void add(String contextName, String type, PropertyBags objects);
+    void add(String contextName, String namespace, String type, PropertyBags objects);
+
+    String add(String contextName, String namespace, String type, PropertyBag properties);
```

Comment:
Maybe we should add some javadoc about the returned value, it's not obvious in the API.

Reply:
I wonder : is it the responsibility of the triple store to decide of the ID of added resources ?
(anyway for this PR let's leave it this way)

Reply:
Javadoc added.
In this methods the ids of the created resources are defined inside the add methods (there is no need to provide the ids as input).
We could, in future developments, add a method that gets the id(s) as input (it could be necessary, for having a full CGMES export).